scripts/install

```diff
@@ -132,23 +146,54 @@ ynh_replace_string --match_string="yunobase" --replace_string="$db_name" --targe
 ynh_replace_string --match_string="yunomail" --replace_string="$email" --target_file="$config"
 ynh_replace_string --match_string="yunodomain" --replace_string="$domain" --target_file="$config"
 ynh_replace_string --match_string="language" --replace_string="$language" --target_file="$config"
+# Enable or disable DAV support for users
+if [ $dav_support -eq 0 ]
+then
+	ynh_replace_string --match_string="__DAV__" --replace_string="false" --target_file="$config"
+	dav="Yes"
+else
+	ynh_replace_string --match_string="__DAV__" --replace_string="true" --target_file="$config"
+	dav="No"
+fi
+# Enable or disable signup for public users
+if [ $signup -eq 1 ]
+then
+	ynh_replace_string --match_string="__SIGNUP__" --replace_string="false" --target_file="$config"
+	sign="No"
+else
+	ynh_replace_string --match_string="__SIGNUP__" --replace_string="true" --target_file="$config"
+	sign="Yes"
+fi
+# Enable or disable two factor authentication support for  users
+if [ $two_factor -eq 1 ]
+then
+	ynh_replace_string --match_string="__TWO_FACTOR__" --replace_string="true" --target_file="$config"
+	twof="Yes"
+else
+	ynh_replace_string --match_string="__TWO_FACTOR__" --replace_string="false" --target_file="$config"
+	twof="No"
+fi
```

Comment:
```suggestion
# Enable or disable DAV support for users
if [ $dav_support -eq 0 ]
then
        dav_boolean="false"
	dav="Yes"
else
        dav_boolean="true"
	dav="No"
fi

# Enable or disable signup for public users
if [ $signup -eq 1 ]
then
	sign_boolean="false"
	sign="No"
else
	sign_boolean="true"
	sign="Yes"
fi


# Enable or disable two factor authentication support for  users
if [ $two_factor -eq 1 ]
then
	twof_boolean="true"
	twof="Yes"
else
	twof_boolean="false"
	twof="No"
fi

ynh_replace_string --match_string="__DAV__" --replace_string="$dav_boolean" --target_file="$config"
ynh_replace_string --match_string="__SIGNUP__" --replace_string="$sign_boolean" --target_file="$config"
ynh_replace_string --match_string="__TWO_FACTOR__" --replace_string="$twof_boolean" --target_file="$config"
```

This suggestion defines some `$foobar_boolean` value with true/false to avoid having a gazillion `ynh_replace_string`

Also would be nice to simplify this entire configuration with `ynh_add_config`